tools/alive-tv.cpp

```diff
@@ -314,8 +315,21 @@ static llvm::Function *findFunction(llvm::Module &M, const std::string FName) {
   return 0;
 }
 
-static const char *Usage =
-R"EOF(Alive2 stand-alone translation validator:
+static ofstream OutFile;
+
+int main(int argc, char **argv) {
+  llvm::sys::PrintStackTraceOnErrorSignal(argv[0]);
+  llvm::PrettyStackTraceProgram X(argc, argv);
+  llvm::EnableDebugBuffering = true;
+  llvm::llvm_shutdown_obj llvm_shutdown; // Call llvm_shutdown() on exit.
+  llvm::LLVMContext Context;
+
+  std::string Usage =
+      R"EOF(Alive2 stand-alone translation validator:
+version )EOF";
+  Usage += alive_version;
```

Comment:
It is a really good idea to use the C string here, not a macro, so that version change only causes relinking but not recompilation.